src/styles/partials/components/_icon-heading-dark.scss

```diff
@@ -3,10 +3,10 @@
   .dg_icon-heading__image-container {
     background: none;
 
-    :before,
-    i:before {
-      background: $pale-blue;
-    }
+     :before,
```

Comment:
Question regarding this item. Is the selector intended to be (in CSS format) `.dg_icon-heading__image-container:before`? If so, it should be updated to the following in SCSS: `&:before`. This would apply to any other pseudo-elements in the file as well. Ie. `&after,`.